style_editor.html

```diff
@@ -5,6 +5,8 @@
   <meta charset="UTF-8">
   <!-- This should show up in search engine results. -->
   <meta name="description" content="ProffieOS Style Editor: Customize your Proffieboard saber blade animations with this browser based tool.">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css" crossorigin="anonymous"/>
```

Comment:
I think we should add these files into the git repository instead of accessing them remotely.
This sets up for making it a PWA in the future. (You can't host remote files in a PWA, they must be from the same domain.)

Reply:
ok. Made a separate PR for that change.